observability/akka-http-metrics/src/main/scala/com/daml/metrics/akkahttp/AkkaHttpMetrics.scala

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.metrics.akkahttp
+
+import scala.concurrent.ExecutionContext
+
+import akka.util.ByteString
+import akka.stream.scaladsl.{Source, Flow, Sink}
+import akka.http.scaladsl.model.{RequestEntity, ResponseEntity, HttpEntity}
+import akka.http.scaladsl.server.{Directive, Route}
+import akka.http.scaladsl.server.RouteResult._
+
+import com.daml.metrics.Timed
+import com.daml.metrics.api.MetricsContext
+import com.daml.metrics.api.MetricHandle.{Counter, Timer}
+
+/** Support to capture metrics on akka http
```

Comment:
Nit: I suggest consistent usage of commenting style, i.e., either use `/* */` or `//` throughout this file (the latter being preferred).